common/button/button-en.html

```diff
@@ -0,0 +1,429 @@
+---
+{
+	"title": "Buttons",
+	"language": "en",
+	"altLangPage": "button-fr.html",
+	"description": "Buttons page including working examples to test how various button styles appear and function",
+	"dateModified": "2023-06-14"
+}
+---
+
+<h2>On this page:</h2>
+
+<ul>
+	<li><a href="#buttons/links">Buttons vs links</a></li>
+	<li><a href="#basic">Basic use (Default)</a></li>
+	<!-- <li><a href="#enhanced">Enhanced use</a></li> -->
+	<li><a href="#colours/styles">Colours/Styles</a></li>
+	<li><a href="#calltoaction">Supertask (Call to Action)</a></li>
+	<li><a href="#sizing">Sizing</a></li>
+	<li><a href="#fullwidth">Full Width</a></li>
+	<li><a href="#activestate">Active State</a></li>
+	<li><a href="#disabledstate">Disabled State</a></li>
+	<li><a href="#bghorizontal">Button Groups (Horizontal)</a></li>
+	<li><a href="#bgvertical">Button Groups (Vertical)</a></li>
+	<li><a href="#buttontoolbar">Button Toolbar</a></li>
+	<li><a href="#inputgroups">Input Groups</a></li>
+	<li><a href="#datatoggle">Data-toggle Buttons</a></li>
+	<li><a href="#resource">Resource</a></li>
+</ul>
+
+<h2 id="buttons/links">Buttons vs links</h2>
+<dl class="dl-horizontal">
+<dt>Button</dt>
+<dd>A button will perform an action on the current page without moving the focus or without navigating the user away</dd>
+<dt>Link</dt>
+<dd>A link will send the user at a location, in the same page via an anchor (`#`) link or to an another page without sending any data over.</dd>
+</dl>
+<p>Do not use the role button on an anchor, use a button element instead.</p>
+
+<h2 id="basic">Basic use (Default)</h2>
+
+<h3>Working example</h3>
+<h4>Scaffolding (Default):</h4>
+<button type="submit">Button</button>
+<input type="button" value="Input" />
+<input type="submit" value="Submit" />
+<a href="#" >Link</a>
+
+<h4>With the default style:</h4>
+<button class="btn btn-default" type="submit">Button</button>
+<input class="btn btn-default" type="button" value="Input" />
+<input class="btn btn-default" type="submit" value="Submit" />
+<a class="btn btn-default" href="#" role="button">Link</a>
+
+<h4>Code Sample</h4>
+<pre><code>
+	// Default button:
+	&lt;button type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input type="button" value="Input"/&gt;
+	&lt;input type="submit" value="Submit"/&gt;
+	&lt;a href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+
+	// Styled button:
+	&lt;button class="<strong>btn btn-default</strong>" type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="button" value="Input"/&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="submit" value="Submit"/&gt;
+	&lt;a class="<strong>btn btn-default</strong>" href="#" role="button"&gt;
+	Link
```

Comment:
Please remove the `role=button` on anchor

```suggestion
<a class="btn btn-default" href="#">Link</a>

<h4>Code Sample</h4>
<pre><code>
	// Default button:
	&lt;button type="submit"&gt;
	Button
	&lt;/button&gt;
	&lt;input type="button" value="Input"/&gt;
	&lt;input type="submit" value="Submit"/&gt;
	&lt;a href="#"&gt;
	Link
	&lt;/a&gt;

	// Styled button:
	&lt;button class="<strong>btn btn-default</strong>" type="submit"&gt;
	Button
	&lt;/button&gt;
	&lt;input class="<strong>btn btn-default</strong>" type="button" value="Input"/&gt;
	&lt;input class="<strong>btn btn-default</strong>" type="submit" value="Submit"/&gt;
	&lt;a class="<strong>btn btn-default</strong>" href="#"&gt;
	Link
```

Reply:
Changes made in the commit "Addressed Requested Changes".